import-service/src/functions/importProductsFile/handler.ts

```diff
@@ -0,0 +1,35 @@
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import * as AWS from 'aws-sdk';
+
+export const importProductsFile = async (event) => {
+  console.log('[importProductsFile] called, arguments: ', JSON.stringify(event.queryStringParameters));
+
+  const queryParams = event.queryStringParameters;
+
+  if (!queryParams?.name) {
```

Comment:
you can destructure `name` property from the `event.queryStringParameters` object on the line above

Reply:
Done.